backend/canvas_app_explorer/tests.py

```diff
@@ -1,3 +1,12 @@
 from django.test import TestCase
-
+from .canvas_lti_manager import CanvasLtiManager
+from django.conf import settings
 # Create your tests here.
+class TestCanvasLtiManager(TestCase):
+    def test_get_ex_tools(self):
+        API_URL =  settings.TEST_API_URL
+        API_KEY =  settings.TEST_API_KEY
+        COURSE_ID = settings.TEST_COURSE_ID
+        manager = CanvasLtiManager(API_URL=API_URL, API_KEY=API_KEY, course_id=COURSE_ID)
+        res = manager.get_tools_available_in_course()
+        self.assertEqual(len(res), 10)
```

Comment:
I think maybe we could just check if each of the elements of `res` is an `ExternalTool`. So, using `isinstance(elem, ExternalTool)`. This won't consistently be 10.